frontend/tsconfig.app.json

```diff
@@ -3,7 +3,7 @@
     "extends": "./tsconfig.json",
     "compilerOptions": {
         "outDir": "./out-tsc/app",
-        "types": []
+        "types": ["@angular/localize"]
```

Comment:
Why do we need this?

Reply:
We don't it is a leftover from a different approach that I forgot to remove